### PR TITLE
Add jammy support

### PR DESCRIPTION
**Pull request by WRFitch**
## Proposal
Builds & deploys on Jammy

## Context
- Tests have been moved to Jammy, with no provision for Focal. This is because I'm planning to deprecate focal once Jammy is implemented in the postgresql-bundle repo.

### CODE REVIEW

**Review by marceloneppel — DISMISSED** [review]

File: tests/integration/relations/test_peers.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ async def test_deploy_at_scale(ops_test):
     # Build, deploy, and relate charms.
     charm = await ops_test.build_charm(".")
     async with ops_test.fast_forward():
-        await ops_test.model.deploy(charm, application_name=PGB, num_units=3)
+        await ops_test.model.deploy(charm, application_name=PGB, num_units=3, series="jammy")
```
Comment (marceloneppel) [inline tests/integration/relations/test_peers.py:36]: Is it worth to have something like the [parametrised series](https://github.com/canonical/postgresql-operator/blob/01936670557ba4a65473cc45f1078ae71cbb5cdf/tests/integration/test_charm.py#L32) we had some months ago on the PostgreSQL charm?

**Review by marceloneppel — APPROVED** [review]

File: tests/integration/relations/test_peers.py (hunk 1 above)
Comment (WRFitch) [reply tests/integration/relations/test_peers.py:36]: I had actually planned to remove focal support entirely after this PR was uploaded to charmhub (and therefore usable in the bundle), making multiple series testing unnecessary, but I suppose that's a product decision

**Review by zmraul — APPROVED** [review]

**Review by dragomirp — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Review by taurus-forever — APPROVED** [review]
I am not sure why we need to remove focal, as anyway we need to learn how to support several series in parallel. Anyway, LGTM. Tnx!

**Comment by WRFitch** [comment]
> I am not sure why we need to remove focal, as anyway we need to learn how to support several series in parallel. 

The idea behind supporting Jammy is to allow us to use some newer features of PGB without having to build a snap yet. The reason why I don't want to support both is because I'd still have to figure out a workaround on the focal charm, and it would also slow development since test runs would be doubled across series.

**Pull request merged**